docs/source/intro/get_started.rst

```diff
@@ -0,0 +1,123 @@
+.. _intro/get_started:
+
+****************
+Getting Started
+****************
+
+Installation
+============
+
+There are numerous routes to setting up the full AiiDA environment, depending on your use case.
+Here we first provide the simplest approach for installation on your local computer.
+
+`Conda <https://docs.conda.io>`_  provides a cross-platform package management system, from which we can install the basic components of the AiiDA infrastructure in an isolated environment:
+
+.. code-block:: console
+
+    $ conda create -n aiida -c conda-forge aiida-core aiida-core.services
+    $ conda activate aiida
+    $ reentry scan
+
+.. note::
+
+    You can also install ``aiida-core`` from `PyPi <https://pypi.org/project/aiida-core>`_
+
+This installs the three core components of AiiDA:
+
+* ``aiida-core``: The main python package and associated CLI ``verdi``.
+* `PostgreSQL <https://www.postgresql.org>`_: A service which manages the database where we store generated data.
+* `RabbitMQ <https://www.rabbitmq.com>`_: A service which manages communication with the processes that we run.
+
+To initialise a database cluster with PostgreSQL and start the service:
+
+.. code-block:: console
+
+    $ initdb -D mylocal_db
+    $ pg_ctl -D mylocal_db -l logfile start
+
+We can then use the `quicksetup` command, to set up an AiiDA configuration profile and related data storage.
+
+.. code-block:: console
+
+    $ verdi quicksetup
+    Info: enter "?" for help
+    Info: enter "!" to ignore the default and set no value
+    Profile name: me
+    Email Address (for sharing data): me@user.com
+    First name: my
+    Last name: name
+    Institution: where-i-work
+
+At this point you now have a working AiiDA environment, from which you can add and retrieve data.
+
+.. tip::
+
+    Enable tab completion of ``verdi`` commands in the terminal with:
+
+    .. code-block:: console
+
+        $ eval "$(_VERDI_COMPLETE=source verdi)"
+
+In order to run computations, one additional step is required to start the services that manage these background processes:
+
+.. code-block:: console
+
+    $ rabbitmq-server -detached
```

Comment:
Similar question, make part of `conda activate aiida`?

Reply:
see above comment